index.html

```diff
@@ -94,7 +97,55 @@ <h3>Multi-Post Stories</h3>
                             <li>Ruby</li>
                         </ul>
                     </div>
-                    <button class="btn btn-style" type="button">See Project</button>
+                    <button data-modal-target="#modal" class="btn btn-style" type="button">See Project</button>
+                    <div class="modal" id="modal">
```

Comment:
- [ ] Good job applying the modal here, but please make sure to make it responsive so you can see the modal properly in both mobile and desktop versions. Please apply the necessary changes.